orchestration/runner.go

```diff
@@ -74,10 +76,13 @@ func (r *Runner) runContainer(logger Logger, vd *variantData) error {
 	containerLinks := []string{}
 	for _, service := range servicesList {
 		name := fmt.Sprintf("service-%s-%s-%d", r.context.projectID, r.context.jobID, vd.variant.Number)
```

Comment:
Having more than 1 service will cause all of them to have the same name, which will make the docker daemon reject the run command with a "name already exists" error.

A possible fix:

```go
for sidx, service := range servicesList {
 		name := fmt.Sprintf("service-%s-%s-%d-%d", r.context.projectID, r.context.jobID, vd.variant.Number, sidx)		 		
```